common/views/components/DownloadLink/DownloadLink.tsx

```diff
@@ -52,13 +72,21 @@ const DownloadLink: FunctionComponent<Props> = ({
     href={href}
     onClick={() => {
       trackEvent('download', { width, mimeType, tags: trackingTags });
-      trackGaEvent(trackingEvent);
+      trackingEvent && trackGaEvent(trackingEvent);
     }}
   >
     <span className="flex-inline flex--v-center">
       <Icon icon={download} />
-      <span className="underline-on-hover">{linkText}</span>
-      {format && <Format as="span">{format}</Format>}
+      <TextToDisplay>{linkText || children}</TextToDisplay>
+      {format && (
+        <Space
+          as="span"
+          h={{ size: 'm', properties: ['margin-left'] }}
+          className={`${font('intb', 5)} font-neutral-600`}
```

Comment:
I've merged in a change yesterday where we don't declaire `font-[color]` as a styling class anymore, so better to style it differently using `props.theme.color('neutral.600')` somehow.